release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -863,7 +863,14 @@ Support for the Broadcom BCM57504 network interface controller is now available
 [id="ocp-4-14-networking-ovn-kubernetes-secondary-network"]
 ==== OVN-Kubernetes is available as a secondary network
 
-With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports both layer 2 switched and localnet switched topology networks.For more information about OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.adoc#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network].
+With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports both layer 2 switched and localnet switched topology networks. For more information about OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.adoc#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network].
+
+[id="ocp-4-14-global-ip-forwarding-network-disabled"]
+==== Global IP forwarding is disabled on OVN-Kubernetes based cluster deployments
+
+From this release, global IP address forwarding is disabled on OVN-Kubernetes based cluster deployments to prevent undesirable effects for cluster administrators with nodes acting as routers. OVN-Kubernetes now enables and restricts forwarding on a per-managed interface basis.
```

Comment:
"From this release" - maybe "starting with this release" or "beginning with this release" would be more clear?